### PR TITLE
added i18n and Translated to UA

### PR DESCRIPTION
**Pull request by Nikita88575**

### CODE REVIEW

**Comment by Nikita88575** [comment]
Screenshot at 2024-01-11 11-48-00
Screenshot at 2024-01-11 11-48-26

File: comments/migrations/0003_alter_comment_author_alter_comment_post.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Generated by Django 5.0 on 2024-01-11 00:54
```
Comment (acman) [inline comments/migrations/0003_alter_comment_author_alter_comment_post.py:1]: I don't think you need migrations here. There are some changes from main branch. Merge main and I think you don't need migrations for i18n

**Review by acman — CHANGES_REQUESTED** [review]
Looks really nice! Resolve migrations and it is ready to go

**Review by acman — APPROVED** [review]
👍

**Pull request merged**